docker/hive2-kerberos/files/conf/hive-site.xml

```diff
@@ -100,13 +95,18 @@
     </property>
 
     <property>
-        <name>hive.security.authorization.manager</name>
-        <value>org.apache.hadoop.hive.ql.security.authorization.plugin.sqlstd.SQLStdConfOnlyAuthorizerFactory</value>
+        <name>hive.server2.use.SSL</name>
```

Comment:
V testovacom prostredi som musel spravit nejake zmeny, aby to fungovalo, napr. zapnut SSL.